scripts/rs-enum.sh

```diff
@@ -16,14 +16,30 @@
 #
 # Alderlake:
 #$ ./rs-enum.sh 
```

Comment:
Please add some comment with explanation of the commands sent in this script

Reply:
you are not commenting c++ code for every magic you do.
This is simple filtering, nothing specific can be commented. No magic involved.

Reply:
I think Remi meant command line parameters
i.e. 
```
# parse command line parameters
# for '-i' parameter, print links only
```